src/state/store.ts

```diff
@@ -186,3 +194,28 @@ function deleteConfig() {
 function saveConfig(config: ConnectorConfig) {
   window.localStorage.setItem('bc:config', JSON.stringify(config));
 }
+
+/**
+ * Automatically connect to a user's NWC wallet if the URL hash parameter "nwc"
+ * contains a valid NWC connection URL
+ */
+function autoConnect() {
+  try {
+    const {hash} = window.location;
+    if (hash && hash.length > 150) {
```

Comment:
why this length check? it seems arbitrary and not needed

Reply:
just an additional check not to even try. can be removed..